website/views.py

```diff
@@ -164,21 +209,44 @@ def question_answer(request, question_id):
                 subject, '',
                 sender_email, to,
                 bcc=[bcc_email],
-                headers = {"Content-type":"text/html;charset=iso-8859-1"}
+                headers={"Content-type": "text/html;charset=iso-8859-1"}
             )
             email.attach_alternative(message, "text/html")
             email.content_subtype = 'html'  # Main content is text/html
             email.mixed_subtype = 'related'
-            email.send(fail_silently = True)
+            email.send(fail_silently=True)
+
+            answer_sets = Answer.objects.filter(
```

Comment:
variable name can be ```answers``` instead of answer_sets.
Similarly for the below defined variables.

Reply:
variable has been changed with answers